### DISCUSSION TITLE
Can I integrate elfeed into this?

### DISCUSSION THREAD

**Discussion by port19x** [emacs-dashboard/dashboard#465] [Q&A] [answered]
I'd love to be able to see recent / unread elfeed entries on my dashboard after an implicid `elfeed-update`.
Some ideally optional number of maximum entries could be configured like bookmarks and recents.

Does such a feature exist / is there some known workaround?

**Comment by jcs090218** [comment, marked as answer]
There is a package [elfeed-dashboard](https://github.com/manojm321/elfeed-dashboard). Is this what you are looking for? 🤔
